src/test/java/com/microsoft/azure/storage/ContainerAPITest.groovy

```diff
@@ -1627,6 +1627,12 @@ class ContainerAPITest extends APISpec {
     def "Root explicit"() {
         setup:
         cu = primaryServiceURL.createContainerURL(ContainerURL.ROOT_CONTAINER_NAME)
+        // Create root container if not exist.
+        try {
+            cu.create(null, null, null).blockingGet()
+        }
+        catch (Exception e) {
```

Comment:
Could you catch a StorageException and check e.errorCode == StorageErrorCode.ContainerAlreadyExists, please? And rethrow if the condition fails